apps/website/screens/components/chip/overview/examples/categorization.ts

```diff
@@ -0,0 +1,21 @@
+import { DxcChip, DxcInset, DxcFlex } from "@dxc-technology/halstack-react";
+
+const code = `() => {
+  return (
+    <DxcInset space="2rem">
+      <DxcFlex gap="0.5rem" justifyContent="center">
+        <DxcChip label="GPDR Compliant" suffixIcon="open_in_new" />
```

Comment:
In the context of the example (and also in the notion), the suffixes may be actions.